src/static/javascripts/components/dashboard.jsx

```diff
@@ -121,7 +121,7 @@ var MainDiv = React.createClass({
 		if (!loggedIn()) {
 		   return (<Welcome />);
 		}
-        var itemNodes = testQuestions.map(function (item) {
+        var itemNodes = data.map(function (item) {
```

Comment:
@untra @CarlosHL is data going to be updated live? (data is defined as var data = {% raw survey_json %}; earlier), because if not we'll need a handler for future changes in what we want to display on the dashboard.. right?